share.md

```diff
@@ -1,29 +1,30 @@
 ---
 title: Share
 excerpt: "We have a projects page where users can browse projects that have been built using ev3dev. You can add your own too! Each project gets a dedicated page for the author to explain what they have been working on, as well as provide videos, pictures, build instructions, code, and any other media that pertains to the project."
+extra-head-content: '<script async defer src="https://buttons.github.io/buttons.js"></script>'
 ---
 <div class="lead">
     <center>
         Do you like ev3dev? Show your support!
     </center>
 </div>
 
-<div class="row">
-    <div class="col-md-6" markdown="1">
-<h3><span class="glyphicon glyphicon-star heading-icon"></span>Give us a star on GitHub</h3>
-"Stars" are a great way to show support for a GitHub project. Visit [ours][] and star us!
-</div>
-    <div class="col-md-6" markdown="1">
-<h3><span class="glyphicon glyphicon-check heading-icon"></span>Vote for ev3dev</h3>
-Vote for ev3dev in the [mindsensor.com][] poll (it's about half way down
-the page).
-</div>
-</div>
+Give us a star on GitHub
```

Comment:
I think I probably just normalized to whatever was below it; if the prefix version is preferred (which is what I generally use) I'll switch both out for that.